lib/pbench/test/unit/server/conftest.py

```diff
@@ -10,6 +10,11 @@
 
 [pbench-server]
 pbench-top-dir = {TMP}/srv/pbench
+bycrypt_log_rounds = 5
+token_expiration_duration = 1
+
+[Postgres]
+db_uri = sqlite:///:memory:
```

Comment:
Perhaps put this in a `[Sqlite3]` section?

Reply:
Eventually we would be having a postgres section in our server config file and thats what I was trying to replicate in our `conftest.py` file. However, I am using an in-memory database for the unit testing and I guess there is only `sqlite3` available in python. If I make `sqlite` section here I will have make some changes in code by adding a condition like `if [postgres] not present then check [sqlite]` but if we never gonna have a `sqlite` section this condition would be unnecessary.